package.json

```diff
@@ -500,6 +500,11 @@
                         "description": "Activate keybindings.",
                         "default": true,
                         "scope": "window"
+                    },
+                    "calva.displayDiagnostics": {
+                        "type": "boolean",
+                        "markdownDescription": "Show linter warnings and other diagnostics from clojure-lsp and clj-kondo.",
+                        "default": true
```

Comment:
I think "from clojure-lsp and clj-kondo" can be removed from this description since this could be confusing to a user who may want to use the clj-kondo extension for linting but disable linting from clojure-lsp (had someone mention doing this recently), or a user who doesn't really know/care what clojure-lsp is. For all they may know linting comes from Calva.